Signal/src/view controllers/FullImageViewController.m

```diff
@@ -128,56 +159,106 @@ - (void)populateImageView:(UIImage *)image {
 }
 
 - (void)initializeGestureRecognizers {
-    self.doubleTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(imageDoubleTapped:)];
-    self.doubleTap.numberOfTapsRequired = 2;
-
-    self.singleTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(imageSingleTapped:)];
-    [self.singleTap requireGestureRecognizerToFail:self.doubleTap];
-
-    self.singleTap.delegate = self;
-    self.doubleTap.delegate = self;
+    UITapGestureRecognizer *singleTap = [[UITapGestureRecognizer alloc] initWithTarget:self
+                                                                                action:@selector(imageDismissGesture:)];
+    singleTap.delegate = self;
+    [self.view addGestureRecognizer:singleTap];
+    
+    UITapGestureRecognizer *doubleTap = [[UITapGestureRecognizer alloc] initWithTarget:self
```

Comment:
> Is that persuasive?

I'd assume iMessage knows all of this, and arrived at their conclusion for a reason, but I won't block the PR on it.